packages/office-ui-fabric-react/src/components/FloatingPicker/PeoplePicker/FloatingPeoplePickerPage.tsx

```diff
@@ -27,7 +27,7 @@ export class FloatingPeoplePickerPage extends React.Component<IComponentDemoPage
         propertiesTables={
           <PropertiesTableSet
             sources={ [
-              require<string>('!raw-loader!@uifabric/experiments/src/components/FloatingPicker/BaseFloatingPicker.types.ts')
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/FloatingPicker/BaseFloatingPicker.types.ts')
```

Comment:
intentional?

Reply:
Yes, this is intentional, as I've moved this types file from experiments to OUFR